src/test/test.cpp

```diff
@@ -216,8 +218,9 @@ runTest(CommandLineArgs const& args)
     session.cli(parser);
 
     auto result = session.cli().parse(
-        args.mCommandName, Catch::clara::detail::TokenStream{
-                               std::begin(args.mArgs), std::end(args.mArgs)});
+        fmt::format("{0} {1}", args.mExeName, args.mCommandName),
```

Comment:
same here (spaces in "exeName") - not as critical as it's test code though

Reply:
I switched this one to only use mCommandName, as it contains "stellar-core" inside